<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaModificarTarifa</class>
 <widget class="QMainWindow" name="VentanaModificarTarifa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>350</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modificar Tarifa</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
font: 20pt &quot;Tahoma&quot;;
}

QLineEdit, QPushButton{
font: 20pt &quot;Tahoma&quot;;
border:1px solid black;
border-radius: 5px;
}

#labelModificarTarifa{
font: 40pt &quot;Tahoma&quot;;
}

#labelTarifaActual, #labelNuevaTarifa{
font: 25pt &quot;Tahoma&quot;;
}

#btnGuardar:hover{
background-color: #2aff27;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelModificarTarifa">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>MODIFICAR TARIFA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTarifaActual">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TARIFA ACTUAL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelNuevaTarifa">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>NUEVA TARIFA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>CARRO :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtActualCarro">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtActualMoto">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>MOTO :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>CARRO :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>MOTO :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtNuevoCarro">
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtNuevoMoto">
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>20</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>F:/UQ/2018-1/VIII. Ingenieria de Software III/ProyectoFinal/Parqueadero/images/icon-guardar.png</normaloff>F:/UQ/2018-1/VIII. Ingenieria de Software III/ProyectoFinal/Parqueadero/images/icon-guardar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
